snuba/datasets/sessions_processor.py

```diff
@@ -58,7 +60,7 @@ def process_message(self, message, metadata) -> Optional[ProcessedMessage]:
         processed = {
             "session_id": str(uuid.UUID(message["session_id"])),
             "distinct_id": str(uuid.UUID(message.get("distinct_id") or NIL_UUID)),
-            "quantity": 1,
+            "quantity": quantity,
```

Comment:
I remember at a point we discussed a quite different protocol for pre-aggregated sessions. Was that dropped in favor of just relying on the quantity field? 
Asking because it is essential that the processor is always backward compatible. If processing a message raises an exception, the consumer stops and won't restart autonomously till the message is either removed or the consumer fixed to cope with it.

Reply:
Yes, we decided that its best to keep the kafka-snuba messages as close to the original as possible. The rest is done in relay. Also yes, the consumer can continue ingesting normal individual session payloads.